### PR TITLE
feat:add utility and test

### PR DESCRIPTION
**Pull request by iakivpekarskyi**
1. Tasks's name: 
  [IL-261] Utility for symbol $

2. Developer's full name:
 Iakiv Pekarskyi

3. Changes:

-  Created utility that adds dollar symbol to the number.
-  Added test to check if it works properly.

### CODE REVIEW

File: src/utility/DollarPrice.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { useCombinedStore } from '@/store/store'
+
+export default function DollarPrice() {
```
Comment (Wdyffs) [inline src/utility/DollarPrice.tsx:3]: Name it just Price, because we do not know future currency

File: src/utility/DollarPrice.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+describe('DollarPrice', () => {
+  it('should render the total price in the correct format', () => {
+    render(<DollarPrice />)
+
+    const formattedTotalPrice = new Intl.NumberFormat('en-US', {
+      style: 'currency',
+      currency: 'USD',
+      maximumFractionDigits: 2,
+      minimumFractionDigits: 2,
+    }).format(0) // Assuming initial total price is 0 in your case
```
Comment (Wdyffs) [inline src/utility/DollarPrice.test.tsx:15]: That lines should be removed
In that test u should render Price component with sum prop sum
And in the end of the test u should check that "$sum" is in the document

File: src/utility/DollarPrice.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  const { totalPrice } = useCombinedStore()
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply src/utility/DollarPrice.tsx]: fixed

File: src/utility/DollarPrice.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  const formattedTotalPrice = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    maximumFractionDigits: 2,
+    minimumFractionDigits: 2,
+  }).format(totalPrice || 0)
+
+  console.log('Formatted Total Price:', formattedTotalPrice)
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply src/utility/DollarPrice.tsx]: fixed

File: src/utility/DollarPrice.tsx (hunk 1 above)
Comment (iakivpekarskyi) [reply src/utility/DollarPrice.tsx]: fixed

File: src/utility/DollarPrice.test.tsx (hunk 2 above)
Comment (iakivpekarskyi) [reply src/utility/DollarPrice.test.tsx]: fixed

**Review by Wdyffs — APPROVED** [review]

**Pull request merged**